samples/charts/data-chart/custom-drawing-annotations/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "react-data-chart-axis-locations",
+  "description": "This project provides example of Data Chart Axis Locations using Infragistics React components",
+  "author": "Infragistics",
+  "homepage": ".",
+  "version": "1.4.0",
+  "private": true,
+  "scripts": {
+    "start": "set PORT=4200 && react-scripts --max_old_space_size=10240 start",
+    "build": "react-scripts --max_old_space_size=10240 build ",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject"
+  },
+  "dependencies": {
+    "igniteui-react-core": "18.2.2-beta.0",
+    "igniteui-react-charts": "18.2.2-beta.0",
+	  "igniteui-react-layouts": "18.2.2-beta.0",
+    "igniteui-react-inputs": "18.2.2-beta.0",
```

Comment:
you need to target proget packages for all IG packagese, e.g.
"@infragistics/igniteui-react-charts": "23.2.152"
also, you need to include this package as the 1st item in `dependencies` 
"@infragistics/igniteui-react": "23.2.152",